src/main/java/com/adobe/aem/compgenerator/utils/CommonUtils.java

```diff
@@ -241,7 +257,13 @@ private static BufferedWriter getFileWriterFromTemplate(String path,
             map.put("sightly", StringUtils.uncapitalize(generationConfig.getJavaFormatedName()));
             map.put("slingModel", generationConfig.getProjectSettings().getModelInterfacePackage() + "." + generationConfig.getJavaFormatedName());
             map.put("CODEOWNER", generationConfig.getProjectSettings().getCodeOwner());
-            map.put("YEAR", generationConfig.getProjectSettings().getYear());
+            String year = generationConfig.getProjectSettings().getYear();
```

Comment:
Let's rely on logic within `getYear()` like it was before - I think current logic was to change empty string to current  year, so we can simply update that logic to also check for "current"